### PR TITLE
Fix pylint's raise-missing-from

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: h/cli/commands/shell.py [hunk 1]
```diff
@@ -75,5 +75,7 @@ def shell(config, type_):
 
     try:
         runner(**locals_)
-    except ImportError:
-        raise click.ClickException("The {!r} shell is not available.".format(type_))
+    except ImportError as e:
+        raise click.ClickException(
+            "The {!r} shell is not available.".format(type_)
+        ) from e
```
Comment (jon-betts) [inline h/cli/commands/shell.py:81]: I think having a variable this short is another lint violation. I think `err` does the trick. We might not want to have to go through this again later!

**Review by jon-betts — CHANGES_REQUESTED** [review]
Looks good, but I think having a variable as short as `e` is another lint violation. So we might want to make sure we don't introduce any more just to have to pull them back out.

File: h/oauth/jwt_grant.py [hunk 2]
```diff
@@ -72,8 +72,8 @@ def create_token_response(self, request, token_handler):
 
         try:
             self.validate_token_request(request)
-        except errors.OAuth2Error as e:
-            return headers, e.json, e.status_code
+        except errors.OAuth2Error as err:
```
Comment (marcospri) [inline h/oauth/jwt_grant.py:75]: For the files I touched I changed all of them to `err`

File: h/schemas/forms/accounts/login.py [hunk 3]
```diff
@@ -47,12 +47,12 @@ def validator(self, node, value):
...
         try:
             user = user_service.fetch_for_login(username_or_email=username)
-        except UserNotActivated:
+        except UserNotActivated as err:
             err = colander.Invalid(node)
             err["username"] = _(
                 "Please check your email and open the link " "to activate your account."
             )
-            raise err
+            raise err from err
```
Comment (jon-betts) [inline h/schemas/forms/accounts/login.py:55]: Don't think this is what you meant

**Review by jon-betts — CHANGES_REQUESTED** [review]
Just one wee change where an exception is created inline and we end-up with `raise err from err` otherwise looks good.

Please don't hate me if Pylint actually doesn't allow `err`

File: h/schemas/forms/accounts/login.py (hunk 3 above)
Comment (marcospri) [reply h/schemas/forms/accounts/login.py]: Ops :facepalm:, fixed now :+1:

**Pull request merged**